### DISCUSSION TITLE
Welcome to dbaclone Discussions!

### DISCUSSION THREAD

**Discussion by sanderstad** [sqlcollaborative/dbaclone#111] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PatrickDBA** [comment]
Currently having problems cloning a Database that contains a MEMORY OPTIMIZED DATA filegroup.  It appears to Fail on the vhdx mounting and can't see the virtual Directory *.mem.  Let me know if you want more details.  Regards Patrick.

**Reply by sanderstad** [reply]
That is probably one of those cases that is not yet supported in and I will have to look into that to see if that could even work that way. Could you  give me the errors and maybe printscreens and create an issue for that?
